### PR TITLE
Fix: Recognition of 'Country', 'Belga Keywords' and 'Country Keywords of AFP, ANP, DPA. [SDBELGA-521]

### PR DESCRIPTION
**Pull request by GyanP**
similar to SDBELGA-521, we have to store data for ANP and DPA also.

### CODE REVIEW

**Comment by petrjasek** [comment]
is there any other ticket related to it @GyanP ?

**Comment by GyanP** [comment]
> is there any other ticket related to it @GyanP ?

No, for AFP we already created the PR, and its merged as well and the ticket is closed but as per the comment mentioned in the ticket, the client wants the same changes for ANP and DPA also so created this one.

File: server/belga/io/feed_parsers/belga_dpa_newsml_2_0.py [hunk 1]
```diff
@@ -175,14 +176,18 @@ def parse_content_meta(self, tree, item):
...
+            belga_keywords = get_resource_service('vocabularies').get_items(_id='belga-keywords', qcode=data.upper())
+            if len(belga_keywords) > 0:
+                item.setdefault('subject', []).append(belga_keywords[0])
+            else:
+                # save all other unmapped keywords as subject with scheme original-metadata
+                item.setdefault('subject', []).append({
+                    'name': data,
+                    'qcode': data,
+                    'scheme': 'original-metadata'
+                })
```
Comment (petrjasek) [inline server/belga/io/feed_parsers/belga_dpa_newsml_2_0.py:190]: seems like this part is more or less duplicatedin base parser, can you avoid that duplication?

File: server/belga/io/feed_parsers/belga_newsml_mixin.py [hunk 2]
```diff
@@ -34,3 +34,17 @@ def _get_countries(self, country_code):
             for c in countries
             if c.get('qcode') == country_code.lower() and c.get('is_active')
         ]
+
+    def _get_keywords(self, data):
+        belga_keywords = get_resource_service('vocabularies').find_one(req=None, _id='belga-keywords').get('items', [])
+
+        _belga_keyword_list = [
+            {'name': c['name'], 'qcode': c['qcode'], 'translations': c['translations'], 'scheme': 'belga-keywords'}
```
Comment (petrjasek) [inline server/belga/io/feed_parsers/belga_newsml_mixin.py:42]: why not use the `get_items` method?

Comment (GyanP) [reply server/belga/io/feed_parsers/belga_newsml_mixin.py:42]: my bad, I will update it with get_items.

**Review by petrjasek — APPROVED** [review]

**Pull request merged**